README.md

```diff
@@ -38,6 +38,18 @@ to your Github account:
 You should now receive an email approximately every two weeks containing the changes made to the APIs since the last update.
 
 * Please note that the date shown is the date that the change was deployed to the Sandbox or Production.
+---
+### XX August 2023 (TBC)
+
+The following change was deployed into sandbox:
```

Comment:
This needs to be:   The following changes are now available in sandbox:

While you're there, could you also change the 8 August entry (from a different PR) to say the same thing please?